merger

```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
```

Comment:
lets allow any number of arguments, not just 2

simple syntax approach
```
def combine_chances(changes: list[float]):

combine_chances([0.2, 0.1, 0.445])
```

or advanced syntax approach:

```
def combine_chances(*args):

combine_chances(0.2, 0.1, 0.445)
```